Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerOrganisationType.cshtml

```diff
@@ -0,0 +1,142 @@
+    {
+        backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-organisation-details-summary";
+        Model.RedirectToSummary = true;
+    }
+
+    string IsSelected(string? radioValue)
+    {
+        return Model.BuyerOrganisationType == radioValue ? "checked='checked'" : string.Empty;
+    }
+}
+
+<a href="@backLink" class="govuk-back-link">
+    Back
+</a>
+
+<main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds govuk-body">
+            <partial name="_ErrorSummary" model="@ModelState" />
+            <form method="post">
+                @if (Request.Query.ContainsKey("frm-summary"))
+                {
+                    <input type="hidden" name="RedirectToSummary" value="true" />
+                }
+                <div class="govuk-form-group @(organisationTypeHasError ? "govuk-form-group--error" : "")">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-fieldset__heading">
+                                @ViewData["Title"]
+                            </h1>
+                        </legend>
+
+                        @if (organisationTypeHasError)
+                        {
+                            <p id="organisationType-error" class="govuk-error-message">
+                                <span class="govuk-visually-hidden">Error:</span> @Html.ValidationMessageFor(m => m.BuyerOrganisationType)
+                            </p>
+                        }
+
+                        <div class="govuk-radios" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="ca-type" name="BuyerOrganisationType"
+                                       type="radio" value="CentralGovernment"
+                                       aria-describedby="ca-type-item-hint" @IsSelected("CentralGovernment")>
+                                <label class="govuk-label govuk-radios__label" for="ca-type">
+                                    @SupplierToBuyerOrganisationTypeModel.BuyerTypes["CentralGovernment"]
+                                </label>
+                                <div id="ca-type-item-hint" class="govuk-hint govuk-radios__hint">
+                                    Includes arm's-length bodies:
+                                    <ul>
+                                        <li>executive agencies (EA)</li>
+                                        <li>non-departmental public bodies (NDPB)</li>
+                                        <li>non-ministerial departments (NMD)</li>
+                                        <li>Trading Funds</li>
+                                    </ul>
+                                </div>
+                            </div>
```

Comment:
As with the other example, please make the first id attribute on the input the same as the name on the model, so that the error summary links come to here. The ids of the others after that are less important, it's the first one that is key

Reply:
@andymantell  I have updated the Ids and Fors
id="@nameof(Model.BuyerOrganisationType)"
for="@nameof(Model.BuyerOrganisationType)"